src/sdk/resources/resource.ts

```diff
@@ -29,6 +29,7 @@ export function getPagedResponse<T>(response: any, items: T[]): PagedResult<T> {
 		hasMore: Boolean(meta && meta.has_more),
 		next: parseLink(meta && meta.links && meta.links.next),
 		previous: parseLink(meta && meta.links && meta.links.prev),
+		downloadLink: meta && meta.links && meta.links.download && meta.links.download.href,
```

Comment:
Should you just save all links rather than looking for this specific link? Especially since it is only valid for certain results.

Reply:
will do!